quality_of_service_demo/rclcpp/src/utils.hpp

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef UTILS_HPP_
```

Comment:
Should this be `QUALITY_OF_SERVICE_DEMO_CPP__UTILS_HPP_`?

Reply:
I thought it should be, too, but the linter complained about it.

Reply:
I guess it's because it's not in the include directory.